src/Seabug/Contract/MarketPlaceListNft.purs

```diff
@@ -68,6 +68,6 @@ marketPlaceListNft = do
         -- I put too low. The old nfts are caught above because their
         -- metadata won't be parsed.
         guard $ (unwrap metadata.seabugMetadata # _.ownerPrice) >=
-          (Natural.fromInt' (10 * 1000000))
+          (Natural.fromBigInt' minAdaOnlyUTxOValue)
```

Comment:
Can we verify a 2ada nft can still be bought?

Reply:
Confirmed that a `minAdaOnlyUTxOValue` (actually 1 ada) priced nft can be bought, this is the purchase transaction: https://testnet.cardanoscan.io/transaction/dace69f58dc611579bd5c7a7cf0de1d4b307f11f9fce01663bc7799cc6fd799c

Reply:
neato!